modules/ROOT/pages/addition/agent-installation/self-registered.adoc

```diff
@@ -0,0 +1,169 @@
+= Self-registering agent
+:description: this page describes NOM agent self-registering setup.
+
+Agent self registration omits manually aquiring initial configuration from NOM UI as described in xref:./manual.adoc#register[Manually registering agent] and instead relies on additional agent configuration on Neo4j instance host to obtain initial configuration from NOM server either through mutual authentication or manual authorization of the agent from UI after installation.
+
+A self-registering agent is useful in an automated environment such as a Kubernetes cluster of Neo4j instances where instances are dynamically created and managed.
+
+[[configure]]
+== Configuration for agent self registration with manual approval
+Following are the required environment variables to be configured to enable self-registration of NOM agent:
+
+[cols="<,<,<",options="header"]
+|===
+| Variable
+| Description
+| Example
+
+| `CONFIG_SERVER_GRPC_ADDRESS`
+| Server GRPC Address
+| server:9090
+
+| `CONFIG_SERVER_HTTP_ADDRESS`
+| Server HTTP address
+| +++https://server:8080+++
+|===
+
+With only the above configuration, agent attempts to connect to NOM server for self-registration which results in agent going into *Standby mode* until authorized.
+
+IMPORTANT: *Standby mode* - this describes the runtime state of a self registering agent which is not authorized yet to monitor an instance. In this mode agent keeps checking with NOM server for approval status at every preset time interval (__default 30 seconds__).
+
+An agent in standby mode shows up in the list of agents in NOM UI. To enable the agent to continue it's normal execution, agent needs to be approved from NOM UI as shown below:
+
+image::agents.png[width=800]
+
+To approve the agent click on `...` agent action icon and click `Approve`:
+
+image::agents.png[width=800]
+
+An edit view to optionally update agent name or description will show as below:
```

Comment:
Does this have to be done before approval to be correctly persisted or will it still be persisted if it's done later?

If it can be done later - I think we should move this step to the end to keep the flow simple.

Reply:
it can be done anytime

Reply:
I think we should remove this step from the instruction in the interest of brevity - would think that the ability to change the name would be obvious enough.